ReactNativeClient/lib/components/screens/note.js

```diff
@@ -979,7 +991,22 @@ class NoteScreenComponent extends BaseScreenComponent {
 				// a plain TextInput for now.
 				// See https://github.com/laurent22/joplin/issues/3041
 				(
-					<TextInput autoCapitalize="sentences" style={this.styles().bodyTextInput} ref="noteBodyTextField" multiline={true} value={note.body} onChangeText={text => this.body_changeText(text)} blurOnSubmit={false} selectionColor={theme.textSelectionColor} keyboardAppearance={theme.keyboardAppearance} placeholder={_('Add body')} placeholderTextColor={theme.colorFaded} />
+					<TextInput
+						autoCapitalize="sentences"
+						style={this.styles().bodyTextInput}
+						ref="noteBodyTextField"
+						multiline={true}
+						value={note.body}
+						onChangeText={text => this.body_changeText(text)}
+						onSelectionChange={({ nativeEvent: { selection } }) => {
+							Object.assign(this.selection, selection);
```

Comment:
Why not `this.selection = selection`?

Reply:
Yeah, not sure. As you can see from the conversation above it I did not understand very well what Object.assign does.
Forgot to fix this one.
Fixed now.